_posts/2018-08-20-grpc_on_http2.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: gRPC on HTTP/2: Engineering a Robust, High Performance Protocol
+published: true
+permalink: blog/grpc_on_http2
+author: Jean de Klerk
+author-link: https://github.com/jadekler
+company: Google
+company-link: https://www.google.com
+---
+In a [previous article](https://grpc.io/blog/http2_smarter_at_scale), we explored how HTTP/2 dramatically increases network efficiency and enables real-time communication by providing a framework for long-lived connections. In this article, we’ll look at how gRPC builds on HTTP/2’s long-lived connections to create a performant, robust platform for inter-service communication. [gRPC](http://grpc.io/) is a high-performance, open-source universal RPC framework. We will explore the relationship between gRPC and HTTP/2, how gRPC manages HTTP/2 connections, and how gRPC uses HTTP/2 to keep connections alive, healthy, and utilized.
```

Comment:
Did you mean to use the unicode `’` or `'` ?

Reply:
As is.